test/e2e/mcd_test.go

```diff
@@ -964,10 +958,6 @@ func TestMCDRotatesCerts(t *testing.T) {
 	}); err != nil {
 		t.Errorf("Mismatch between on disk cert and in cluster cert: %v", err)
 	}
-
-	err = helpers.WaitForCertStatusToChange(t, cs, oldData)
-	require.Nil(t, err)
-
```

Comment:
Since we aren't pausing anymore, can we take out the pause comment at the function header?